include/fmt/format-inl.h

```diff
@@ -8,13 +8,15 @@
 #ifndef FMT_FORMAT_INL_H_
 #define FMT_FORMAT_INL_H_
 
-#include <algorithm>
+#ifndef FMT_IMPORT_STD
+#  include <algorithm>
+#  include <climits>
```

Comment:
Here and elsewhere: `climits` only exports macros so it should be moved outside of the ifndef block.